samples/unit-tests/tooltip/zindex/demo.js

```diff
@@ -0,0 +1,83 @@
+function test(key, options) {
```

Comment:
Polluting the global namespace, wrap the whole content of this file in an iife or a QUnit.module.